package.json

```diff
@@ -39,7 +39,7 @@
     "object-fit-videos": "^1.0.3",
     "ophan-tracker-js": "1.3.27",
     "preact": "^10.5.13",
-    "prebid.js": "https://github.com/guardian/Prebid.js#a6d34e7",
+    "prebid.js": "https://github.com/guardian/Prebid.js#5b7d7b2aeb799ef1e9d43d388a662457c8bdbc1f",
```

Comment:
FYI, hashes only need about 6-8 characters to be unique, so the below would be equivalent.

```suggestion
    "prebid.js": "https://github.com/guardian/Prebid.js#5b7d7b2a",
```

Reply:
Ah good point. I'll make the change

Reply:
I just like to keep lines shorter than 80 characters, but it’s no requirement 😃